KlockanAPI/src/KlockanAPI.Infrastructure/Data/KlockanContext.cs

```diff
@@ -172,5 +172,42 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             SessionDuration = 90,
             CreatedAt = new DateTime(2024, 1, 23, 0, 0, 0, DateTimeKind.Utc)
         });
+
+        // Seed data for Program
```

Comment:
Update comment

Reply:
Fixed in commit [f672744](https://github.com/JU-DEV-LatamBootcamp/Klockan-Backend/commit/f672744c06d72641b52d3612f2253716acd18202).